Automattic/jetpack
# i18n: Prepare At A Glance page for translations.

This wraps everything on the "At A Glance" page, as well as the Support Card in the i18n function, to prep for translations.  

to test: 
- Run `npm run build`
- Run `npm run build-i18n`
- Check `_inc/jetpack-strings.php` to make sure that the strings on At A Glance and Support card
- Make sure all links and stuff still works.
